internal/providers/terraform/aws/dynamodb_table.go

```diff
@@ -40,6 +40,13 @@ func NewDynamoDBTableResource(d *schema.ResourceData) schema.CoreResource {
 		targets = append(targets, newAppAutoscalingTarget(ref, ref.UsageData))
 	}
 
+	pitr := &aws.PointInTimeRecovery{Enabled: false}
+	if d.Get("point_in_time_recovery").Exists() {
+		pitr = &aws.PointInTimeRecovery{
+			Enabled: d.Get("point_in_time_recovery").Array()[0].Get("enabled").Bool(),
```

Comment:
probably want to check the length here - this could cause exceptions with malformed code (often see with vscode)

Reply:
Yup, will do. Was relying on the length check inside `Exists`, but now checking explicitly